Formula/percona-server@5.5.rb

```diff
@@ -1,10 +1,9 @@
 class PerconaServerAT55 < Formula
   desc "Drop-in MySQL replacement"
-  homepage "https://www.percona.com/"
-  url "https://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.41-37.0/source/tarball/percona-server-5.5.41-37.0.tar.gz"
-  version "5.5.41-37.0"
-  sha256 "4de65ccbdd6c266f18339c2ea5427a15d90a8ce1ce1c7574aa2e72f685a10833"
-  revision 1
+  homepage "https://www.percona.com"
```

Comment:
It was removed by mistake, so can re-add it. Just curious why is it so important to have a `/` at the end?